package.json

```diff
@@ -12,6 +12,7 @@
     "size": "lerna run size --stream --parallel",
     "test": "run-s test:*",
     "prepublishOnly": "yarn clean && yarn && yarn build && yarn test && cp README.md packages/styled-components",
+    "postinstall": "yarn bs && yarn build",
```

Comment:
Does this run on normal install as a dependency as well? We only want it to run for local installations.

Reply:
good catch. yeah I believe this is wrong, let me take a look

Reply:
I believe I've fixed this by moving to `prepare`